CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased]
 
+## [1.2.6] - 2022-02-12
```

Comment:
@abuzar347 release date for 1.2.6 is 2022-04-22
Also, match the format of the last two releases to the order releases and update fixes as well.